src/bp/core/routers/admin/server.ts

```diff
@@ -175,7 +176,9 @@ export class ServerRouter extends CustomRouter {
     this._rebootServer = await this.jobService.broadcast<void>(this.__local_rebootServer.bind(this))
   }
 
-  private __local_rebootServer() {
-    process.send && process.send({ type: 'reboot_server' })
+  private async __local_rebootServer(hostname?: string) {
```

Comment:
Is async needed here?

Reply:
Nop you are right, there was something else that was removed. Updated